python_modules/libraries/dagster-datahub/dagster_datahub/resources.py

```diff
@@ -0,0 +1,82 @@
+from typing import Dict
+from datahub.emitter.rest_emitter import DatahubRestEmitter
+from datahub.emitter.kafka_emitter import (
+    DatahubKafkaEmitter,
+    KafkaEmitterConfig,
+    DEFAULT_MCE_KAFKA_TOPIC,
+    MCE_KEY,
+    MCP_KEY,
+    DEFAULT_MCP_KAFKA_TOPIC,
+)
+
+from dagster import resource, InitResourceContext, Noneable, Field, Shape, Array, Map
+
+
+@resource(
+    config_schema={
+        "connection": Field(
+            str, description="Datahub GMS Server", default_value="http://localhost:8080"
```

Comment:
We might want this to be a `StringSource` so that this can be configured with env vars in addition to config.

Also, maybe it's better to not set a default value to a local uri?

Reply:
Assuming the rest emitter should also be StringSource and default removed?

Reply:
Yes, and probably the same for the registry URL?